diff -Nur openscap-1.3.2_old/src/OVAL/oval_variable.c openscap-1.3.2/src/OVAL/oval_variable.c
--- openscap-1.3.2_old/src/OVAL/oval_variable.c	2020-03-11 04:30:50.534661136 -0400
+++ openscap-1.3.2/src/OVAL/oval_variable.c	2020-03-11 04:32:05.787195584 -0400
@@ -156,11 +156,11 @@
 	return oval_collection_iterator_remaining((struct oval_iterator*)iter);
 }
 
-struct oval_variable_possible_restriction *oval_variable_possible_restriction_new(oval_operator_t operator, const char *hint)
+struct oval_variable_possible_restriction *oval_variable_possible_restriction_new(oval_operator_t opera, const char *hint)
 {
 	struct oval_variable_possible_restriction *pr;
 	pr = malloc(sizeof(oval_variable_possible_restriction_t));
-	pr->operator = operator;
+	pr->operator = opera;
 	pr->hint = oscap_strdup(hint);
 	pr->restrictions = oval_collection_new();
 	return pr;
diff -Nur openscap-1.3.2_old/src/OVAL/public/oval_definitions.h openscap-1.3.2/src/OVAL/public/oval_definitions.h
--- openscap-1.3.2_old/src/OVAL/public/oval_definitions.h	2020-03-11 04:30:50.546661222 -0400
+++ openscap-1.3.2/src/OVAL/public/oval_definitions.h	2020-03-11 04:31:30.354943942 -0400
@@ -1669,7 +1669,7 @@
  * @param hint A short description of what the value means or represents.
  * @memberof oval_variable_possible_restriction
  */
-OSCAP_API struct oval_variable_possible_restriction *oval_variable_possible_restriction_new(oval_operator_t operator, const char *hint);
+OSCAP_API struct oval_variable_possible_restriction *oval_variable_possible_restriction_new(oval_operator_t opera, const char *hint);
 
 
 /**
